cmd/runtimetest/main.go

```diff
@@ -643,11 +634,6 @@ func mountMatch(configMount rspec.Mount, sysMount *mount.Info) error {
 }
 
 func validateMounts(spec *rspec.Spec) error {
-	if runtime.GOOS == "windows" {
-		logrus.Warnf("mounts validation not yet implemented for OS %q", runtime.GOOS)
```

Comment:
We want to keep this.  [`mounts` is specified for all platforms][1], and we just aren't testing the Windows-specific parts of that yet.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#mounts

Reply:
I think this can  change to `validatePosixMounts`. Because this is only validated against [POSIX-platform Mounts](https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#posix-platform-mounts).

Reply:
> I think this can change to `validatePosixMounts`. Because this is only validated against [POSIX-platform Mounts][1].

It's also [checking the mount order][2], which [is a cross-platform requirement][3].  Windows checks for mount order could involve extending `cmd/runtimetest/mount` to support Windows, in which case we'd want to keep this valdation generic, with per-platform guards for the [POSIX-specific `type` checks][1], etc.  Or we can duplicate the order checks in a separate `validateWindowsMounts`, in which case calling all of the current logic `validatePosixMounts` is fine.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#posix-platform-mounts
[2]: https://github.com/opencontainers/runtime-tools/blob/eead1367811940eef85d4d953e5435abd2bccdd4/cmd/runtimetest/main.go#L685
[3]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#mounts